### DISCUSSION TITLE
Parcel build fails with Bootstrap 5.3.0-alpha1

### DISCUSSION THREAD

**Discussion by benpotter** [parcel-bundler/parcel#8888] [Q&A]
I'm including the latest version of Bootstrap in my 11ty project and noticed the compiling was failing due to a pseudoelement not being a valid selector in the latest version of Bootstrap (5.3.0-alpha1). Supposedly this has been fixed per this thread https://github.com/twbs/bootstrap/issues/37866

I'm not seeing this changed merged in my npm package so I just made the suggested change to Bootstrap's scss file _floating-lables.scss to fix the issue. This works. However when deploying this project to Netlify, Parcel appears to be failing on the exact same issue (see screenshot below). Not sure how this is happening since I changed the npm Bootstrap source... wouldn't this change persist downstream?

<img width="1195" alt="Screen Shot 2023-03-15 at 07 36 20" src="https://user-images.githubusercontent.com/4133511/225406432-fe5e3820-8779-4840-822b-c9a1de6495fc.png">

Suggestions on how to fix other than changing the version of Bootstrap included in the Parcel build?

**Comment by benpotter** [comment]
Apparently this will be fixed in Bootstrap 5.3.0-alpha2 which should come out in a number of days. Is this really a showstopper? Disappointing that I have to roll back to the 5.2.3 stable version...

**Reply by mischnic** [reply]
Adding this to your package.json might make it ignore that code
```
{
  "@parcel/transformer-css": {
    "errorRecovery": true
  }
}
```
